README.md

```diff
@@ -47,6 +47,31 @@ Currently, the gem supports: `PostgreSQL`.
 
 This configuration provides an example of the structure a PostgreSQL shared storage would require for this purpose. The exact structure should be tailored to the nature of the implemented use cases.
 
+### Optional Dependencies
+
+BAS supports integrations with various external services like GitHub, Notion, OpenAI, and more. To use these integrations, you need to install the required gems manually.
+
+### Installing Optional Dependencies
```

Comment:
What do you think about using a bold title instead of a h3, so the installation details are part of the **Optional dependencies** section?
```suggestion
**Installing Optional Dependencies**
```